src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -140,6 +129,14 @@ private bool ValidateOptions(FileWorkItemsOptions options)
 
             valid = ValidateOutputFileOptions(options) && valid;
 
+            if (options.Inline)
```

Comment:
>options [](start = 16, length = 7)

This has to be done _after_ `ValidateOutputFileOptions`, because that method insists that exactly one of `options.OutputFilePath` and `options.Inline` be set. This is another example of non-uniformity among the command implementations: this is the only command that requires that.